src/inverseRobot.js

```diff
@@ -6,8 +6,36 @@
  * @return {object}
 */
 
+/*
 function inverseRobot(robot) {
```

Comment:
code in github should be clean and without comments) just delete this)

Reply:
> code in github should be clean and without comments) just delete this)

thanks for review, I just didn't notice that:)